lib/publiccloud/provider.pm

```diff
@@ -187,6 +188,20 @@ sub create_ssh_key {
     }
 }
 
+=head2 place_ssh_config
+
+Creates ~/.ssh/config file with all the common ssh client settings
+asdf
```

Comment:
"asdf" ?

Reply:
:smile: I do this so I can quickly jump there. I'll remove it.